Math.md

```diff
@@ -0,0 +1,96 @@
+##Float and Double
+You've already learned about the String and Int types; Strings are values like "hello" or "eat your vegetables", and Ints are whole number values like 34 and 9561.  
+
+Strings may form the backbone of displaying instructions and values to the user, but numbers (and especially mathematical operations on those numbers) are necessary for actually drawing that text to the screen, as well as pretty much every thing the operating system and your applications do, from computing how many days are left in a subscripton to centering a button in window.  
+
+Another kind of number are Float and Double, which represent decimal values such as 0.5 or 3.14159.  The difference between Float and Double is magnitude and precision: Floats are represented in 32 bits and Doubles are represented in 64 bits, so Doubles have the potential for greater magnitude and precision.
+
+You can recognize Floats and Doubles by value because they will have a decimal point; the Swift compiler will interpret any number with a decimal point as a Double unless you specify that it should be recognized as a Float.  Any number without a decimal point will be interpreted by the compiler as an Int.
+
+##Mathematical Operations
+The four primary operators in math are addition (+), subtraction (-), multiplication (*), and division (/); the fifth operator is remainder (%).  For example:
+
+```Swift
+5 / 3 = 1
+5 % 3 = 2
+````
+ 
+ Similarly, you can assign Int and Float values to variables and perform math operations.  For example:
+
+ ```Swift
+ let a = 4
+ let b = 12
+ let n = b / a   (3)
+ let r = b % a   (0)
+ let product = a * b   (48)
+ let sum = a + b   (16)
+ let difference = b - a   (8)
+ ````
+
```

Comment:
This is good. I would use the comment lines // instead of ( ) next to these variables like so:

```swift
let a = 4  // 4

let b = 12  // 12

let n = b / a  // 3

let r = b % a  // 0

let product = a * b  // 48

let sum = a + b  // 16

let difference = b - a  // 8
```